apps/site/lib/site_web/views/layout_view.ex

```diff
@@ -41,11 +41,20 @@ defmodule SiteWeb.LayoutView do
   def nav_link_content(conn),
     do: [
       {"Getting Around", "Transit Services, Plan Your Journey, Riding...",
-       static_page_path(conn, :getting_around)},
+      static_page_path(conn, :getting_around)},
       {"Fares", "Fares By Mode, Reduced Fares, Passes...", cms_static_page_path(conn, "/fares")},
       {"Contact Us", "Phone And Online Support, T-Alerts", customer_support_path(conn, :index)},
       {"More", "About Us, Business Center, Projects...", static_page_path(conn, :about)}
     ]
+    # When time to remove deskstop_nav_redesign feature flag replace the above with below
```

Comment:
You actually can use `Laboratory.enabled?(conn, :deskstop_nav_redesign)` in here too! And then use the result to conditionally show one set of links or the other. :)